core/dom/domobject.js

```diff
@@ -126,15 +126,24 @@ CKEDITOR.dom.domObject.prototype = ( function() {
 		 * references left after the object is no longer needed.
 		 */
 		removeAllListeners: function() {
-			var nativeListeners = this.getCustomData( '_cke_nativeListeners' );
-			for ( var eventName in nativeListeners ) {
-				var listener = nativeListeners[ eventName ];
-				if ( this.$.detachEvent )
-					this.$.detachEvent( 'on' + eventName, listener );
-				else if ( this.$.removeEventListener )
-					this.$.removeEventListener( eventName, listener, false );
-
-				delete nativeListeners[ eventName ];
+			try {
+				var nativeListeners = this.getCustomData( '_cke_nativeListeners' );
+				for ( var eventName in nativeListeners ) {
+					var listener = nativeListeners[ eventName ];
+					if ( this.$.detachEvent ) {
+						this.$.detachEvent( 'on' + eventName, listener );
+					} else if ( this.$.removeEventListener ) {
+						this.$.removeEventListener( eventName, listener, false );
+					}
+
+					delete nativeListeners[ eventName ];
+				}
+			// Catch Edge `Permission denied` error which occurs randomly. Since the error is quite
+			// random, catching allows to continue the code execution and cleanup.
+			} catch ( error ) {
+				if ( !CKEDITOR.env.edge || error.number !== -2146828218 ) {
```

Comment:
Maybe extract issue about this weird Edge error to a new issue? 🤔 We can investigate it further when we find some time.